docs/create-packages/Sign-a-Package.md

```diff
@@ -0,0 +1,101 @@
+---
+title: How to Create a Localized NuGet Package | Microsoft Docs
+author: rido-min
+ms.author: rido-min
+manager: unniravindranathan
+ms.date: 02/23/2018
+ms.topic: article
+ms.prod: nuget
+ms.technology: null
+description: Explains how signed packages can be used to enable content integrity verification
+keywords: NuGet package signing, NuGet security, creating signed packages
+ms.reviewer:
+- karann-msft
+- anangaur
+---
+
+# Signed packages
+
+NuGet packages can include a digital signature which provides protection against tampered content. This signature is produced from an x509 certificate that also adds authenticity proofs to the actual origin of the package. 
+
+> [!Warning] 
+> NuGet.org does not accept signed packages yet. This feature will be announced using the [announcements](https://github.com/NuGet/Announcements) github repository.
+
+## Types of signed packages
+
+NuGet supports two types of signatures:
+
+- **Author Signatures** are created by the package author before the package is being submitted to any repository. These signatures provide validation end to end, from authors to consumers independent from which repository the package has been obtained.
+- **Repository Signatures** provide an integrity guarantee for all packages in a repository whether they are author signed or not, even if those packages are obtained from a different location than the original repository where they were signed.
+
+Considering author and repository signatures there are four possible package signing configurations; no other configurations of these signature types are allowed:
```

Comment:
I think you can simplify this to be just "There are four allowed package signing configurations:"